frontend/integration-tests/tests/monitoring.scenario.ts

```diff
@@ -193,32 +193,227 @@ describe('Monitoring: Silences', () => {
   });
 });
 
-describe('Monitoring: YAML', () => {
+describe('Alertmanager: YAML', () => {
   afterEach(() => {
     checkLogs();
     checkErrors();
   });
 
-  it('displays the YAML page', async () => {
+  it('displays the Alermanager YAML page', async () => {
     await sidenavView.clickNavLink(['Administration', 'Cluster Settings']);
     await crudView.isLoaded();
     await horizontalnavView.clickHorizontalTab('Global Configuration');
     await crudView.isLoaded();
     await monitoringView.wait(
-      until.elementToBeClickable(monitoringView.firstListLinkById('alertmanager')),
+      until.elementToBeClickable(monitoringView.firstElementByTestId('alertmanager')),
     );
-    expect(monitoringView.firstListLinkById('alertmanager').getText()).toContain('Alertmanager');
-    await monitoringView.firstListLinkById('alertmanager').click();
+    expect(monitoringView.firstElementByTestId('alertmanager').getText()).toContain('Alertmanager');
+    await monitoringView.firstElementByTestId('alertmanager').click();
     await crudView.isLoaded();
     await horizontalnavView.clickHorizontalTab('YAML');
     await crudView.isLoaded();
     expect(monitoringView.alertManagerYamlForm.isPresent()).toBe(true);
   });
 
-  it('saves alert-manager.yaml', async () => {
+  it('saves Alertmanager YAML', async () => {
     expect(monitoringView.successAlert.isPresent()).toBe(false);
     await monitoringView.saveButton.click();
     await crudView.isLoaded();
     expect(monitoringView.successAlert.isPresent()).toBe(true);
   });
 });
+
+describe('Alertmanager: Configuration', () => {
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  it('displays the Alermanager Configuration Overview page', async () => {
+    await sidenavView.clickNavLink(['Administration', 'Cluster Settings']);
+    await crudView.isLoaded();
+    await horizontalnavView.clickHorizontalTab('Global Configuration');
+    await crudView.isLoaded();
+    await monitoringView.wait(
+      until.elementToBeClickable(monitoringView.firstElementByTestId('alertmanager')),
+    );
+    expect(monitoringView.firstElementByTestId('alertmanager').getText()).toContain('Alertmanager');
+    await monitoringView.firstElementByTestId('alertmanager').click();
+    await crudView.isLoaded();
+    expect(monitoringView.alertRoutingHeader.getText()).toContain('Alert Routing');
+  });
+
+  it('launches Alert Routing modal, edits and saves correctly', async () => {
+    await crudView.isLoaded();
+    expect(monitoringView.alertRoutingEditButton.isPresent()).toBe(true);
+    await monitoringView.alertRoutingEditButton.click();
+    await crudView.isLoaded();
+
+    expect(monitoringView.firstElementByTestId('input-group-by').isPresent()).toBe(true);
```

Comment:
I think you can skip the `isPresent` checks here. The `sendKeys` fail if not present and we're not waiting for presence, so the checks aren't necessary.

Reply:
fixed